spec/requests/api/internal/v2/court_applications_request_spec.rb

```diff
@@ -9,10 +9,49 @@
   let(:court_application_id) { SecureRandom.uuid }
 
   describe "GET /api/internal/v2/court_applications/{court_application_id}" do
-    it "return 200 succeess" do
+    before do
+      stub_request(:get, "#{ENV['COMMON_PLATFORM_URL']}/applications/#{court_application_id}")
+        .to_return(
+          status: status,
+          headers: { content_type: "application/json" },
+          body: body,
+        )
```

Comment:
I'm okay with your explicit approach, 
anyway I noticed that in this project they are stubbing the calls inside the `webmock.rb` so they can be reused in other tests when.

See the :stub_case_search_with_urn and :stub_hearing_result

should we use the same approach? ... not sure, up to you :)

Reply:
So it looks like for request specs mostly they're explicitly stubbing http calls in the tests, and webmock is being used in _some_ of the service specs (others are using VCR). I feel like this is probably the most consistent, but the existing codebase is quite inconsistent...

Reply:
👍 
the approach I am trying to follow is:
- webmock/fixtures for the services
- VCR for the request spec